src/pkgx.sh/Hero.tsx

```diff
@@ -1,67 +1,127 @@
-import { Box, InputAdornment, Button, TextField, Typography, Stack, Snackbar, Alert, Tooltip, useMediaQuery, useTheme, Tabs, Tab } from "@mui/material";
-import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
-import React, { useState } from "react";
+import {
```

Comment:
Try not to rewrite unrelated parts of the codebase during PRs unless the PR is titled “tidy” since it makes review more tricky and can make it too easy to miss important changes.